package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "automail",
+  "version": "10.0.0",
+  "description": "An enhancement collection for anilist.co",
+  "author": "hoh",
+  "license": "GPL-3.0-or-later",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hohMiyazawa/Automail.git"
+  },
+  "bugs": {
+    "url": "https://github.com/hohMiyazawa/Automail/issues"
+  },
+  "homepage": "https://github.com/hohMiyazawa/Automail",
+  "scripts": {
+    "lint": "eslint --ignore-pattern \"src/build/automail.js\" \"**/*.js\"",
```

Comment:
Should some of the dependencies, like showdown and purify be excluded from linting too?

Reply:
Probably. I think I'd have to see if I can modify or add additional `--ignore-pattern`s to cover more files that should be excluded. Otherwise, if that's not doable I can wrap them with `/*eslint-disable*/` and `/*eslint-enable*/` inline comments.

Reply:
Fixed by using an `.eslintignore` file
https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file